_maps/map_files/MetaStation/MetaStation.dmm

```diff
@@ -75970,7 +75970,6 @@
 	d2 = 8;
 	icon_state = "4-8"
 	},
-/obj/machinery/atmospherics/pipe/simple/hidden/scrubbers,
```

Comment:
This is likely a contender for `Most runtimes fixed in the least amount of diff`